contracts/margin/external/BucketLender/EthUnwrapperForBucketLender.sol

```diff
@@ -0,0 +1,109 @@
+/*
+
+    Copyright 2018 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.4.24;
+pragma experimental "v0.5.0";
+
+import { WETH9 } from "canonical-weth/contracts/WETH9.sol";
+import { BucketLender } from "./BucketLender.sol";
+import { TokenInteract } from "../../../lib/TokenInteract.sol";
+
+
+/**
+ * @title EthUnwrapperForBucketLender
+ * @author dYdX
+ *
+ * Withdraws WETH from a bucketlender, unwraps it, then sends the eth back to the owner
+ */
+contract EthUnwrapperForBucketLender
```

Comment:
Looks good but why not put all this in `EthWrapperForBucketLender`? That would be a lot clearer I feel